package.json

```
@@ -5,6 +5,7 @@
   "repository": "https://github.com/docs/render-content",
   "license": "MIT",
   "scripts": {
+    "lint": "standard --fix && prettier-standard --format '**/*.js'",
```

```suggestion
    "lint": "standard --fix && prettier-standard",
```

I recently learned that this should do the same, but please try it out too for extra confirmation